test/testdrive/protobuf-multiple-message.td

```diff
@@ -13,8 +13,6 @@
 $ set the-schema
 syntax = "proto3";
 
-import "google/protobuf/timestamp.proto";
```

Comment:
Strange that csr was okay with this and rpk's schema registry isn't, any idea what's going on there?

Reply:
no idea, its entirely possible that redpanda is running a new version, or with different default flags, references and imports working is not something csr shipped originally with